src/features/identity/fetch-profile.ts

```diff
@@ -5,7 +5,11 @@ import {
 } from "@/features/dwn/profile-protocol/profile-protocol";
 import { IdentityAgentManager } from "@/features/identity/IdentityAgentManager";
 
-export const fetchProfile = async (identity: ManagedIdentity) => {
+export type FetchProfileResult = ManagedIdentity & Partial<Profile>;
```

Comment:
When fetching a profile, there may or may not be a record in the DWN containing the customized profile information (currently just contains a `displayName`).

There will be situations where this profile record is not in the DWN yet:
* User has imported an identity into the ID Agent, but DWN sync hasn't completed
* User has imported an identity external to our ID agent, but it never has a profile record created for it in it's DWN

In these cases, we still want to display this identity as a "Profile" in our UI, otherwise the user would never be able to see it.